azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -28,16 +28,37 @@ def load_team_help():
     short-summary: Manage service endpoints/connections
     """
 
+    helps['devops service-endpoint create'] = """
+    type: command
+    short-summary: Create a service endpoint using configuration file.
+    long-summary: You can learn more about this at https://aka.ms/azure-devops-cli-service-endpoint
+    """
+
     helps['devops service-endpoint azurerm'] = """
     type: group
     short-summary: Manage Azure RM service endpoints/connections
     """
 
+    helps['devops service-endpoint azurerm create'] = """
+    type: command
+    short-summary: Create an Azure RM type service endpoint.
+    long-summary: For automation, please set service principal password/secret in
+      AZURE_DEVOPS_EXT_AZURE_RM_SERVICE_PRINCIPAL_KEY environment variable. You can learn
+      more about this at https://aka.ms/azure-devops-cli-service-endpoint
+    """
+
     helps['devops service-endpoint github'] = """
     type: group
     short-summary: Manage GitHub service endpoints/connections
     """
 
+    helps['devops service-endpoint github create'] = """
+    type: command
+    short-summary: Create a GitHub service endpoint.
+    long-summary: For automation purpose set the GitHub PAT token in AZURE_DEVOPS_EXT_GITHUB_PAT
```

Comment:
v.v. nit: maybe the wordings in both should match.

Reply:
@geverghe  To consider the feedback.
+ If we need to update section reference in the link.. in that case we will need 3 different aka.ms links.

Reply:
1.Go ahead with point 1.
2. Didn't quite get point 2; why do we need 3 aka.ms links?